src/main/java/org/jitsi/xmpp/component/ComponentBase.java

```diff
@@ -56,6 +61,12 @@
     private final static String PING_THRESHOLD_PNAME = "PING_THRESHOLD";
 
     /**
+     * The name of the property which configures {@link #processingTimeLimit}.
+     */
+    private final static String PROCESSING_TIME_LIMIT_PNAME
```

Comment:
Minor: I think these should be public as argued here: https://github.com/jitsi/jitsi-videobridge/issues/347